Automattic/jetpack
# Google Analytics: switch the module to use the package

## Proposed changes:
* Switch the Jetpack module "Google Analytics" to use the package.
* Replace the module code with backward compatibility wrappers.

### Other information:


## Jetpack product discussion
pfwV0U-4g-p2

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
1. Connect Jetpack.
2. Go to "Settings -> Traffic", scroll to "Google Analytics" section, purchase the necessary plan.
3. Go back to "Settings -> Traffic", follow the link to Calypso to activate GA.
4. Activate the module, enter a random measurement ID (e.g. `G-12345`).
5. Load the site as a visitor, confirm the Google Analytics code is there and it contains `<script async src='https://www.googletagmanager.com/gtag/js?id=` ("legacy" way).
6. Install and activate WooCommerce. Go back to the Calypso settings and activate the "Enhanced eCommerce", click "Save".
7. Load the site as a visitor, confirm the Google Analytics code is there, and it contains `window.ga = window.ga || function(){` ("universal" way).
